Doc/library/csv.rst

```diff
@@ -443,7 +443,8 @@ read CSV files (assuming they support complex numbers at all).
 .. method:: csvwriter.writerow(row)
 
    Write the *row* parameter to the writer's file object, formatted according to
-   the current dialect.
+   the current dialect, and return the return value of the call to the *write*
+   method of the underlying file object.
```

Comment:
Oh... The code uses an structure field called "writeline" but it stores write() in practice:

* https://github.com/python/cpython/blob/3fe7fa316f74ed630fbbcdf54564f15cda7cb045/Modules/_csv.c#L1241
* https://github.com/python/cpython/blob/3fe7fa316f74ed630fbbcdf54564f15cda7cb045/Modules/_csv.c#L1383

@remilapeyre: Maybe you should write a PR to fix the C code, replace writeline with write :-)

Reply:
Yes, it's a trap! It's a simple fix that is not visible by the user; can I put the change in this PR or do I need to open a new one?

Reply:
> or do I need to open a new one?

Please open a separated PR, it's not directly related. For example, this change should go to 3.7, whereas the C code should only be fixed in master.